src/main/java/org/springframework/samples/petclinic/system/TestController.java

```
@@ -0,0 +1,25 @@
+package org.springframework.samples.petclinic.system;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TestController {
+
+    @GetMapping("/test")
+    public String test(String input) {
+        // This code has potential issues for testing the review system
+        if (input == null) {
+            return null; // Should probably throw exception or return default
```

Should throw IllegalArgumentException instead of returning null